### PR TITLE
Add possibility to combine ptp and a relative lin into a blendable sequence.

### PR DESCRIPTION
**Pull request by DanBmh**

### CODE REVIEW

File: pilz_trajectory_generation/src/joint_limits_container.cpp [hunk 1]
```diff
@@ -100,12 +100,20 @@ bool JointLimitsContainer::verifyVelocityLimit(const std::string &joint_name,
...
 
 bool JointLimitsContainer::verifyPositionLimit(const std::string &joint_name,
-                                                     const double &joint_position) const
-{
-  return (!( hasLimit(joint_name)
-             && getLimit(joint_name).has_position_limits
-             && (joint_position < getLimit(joint_name).min_position
-                || joint_position > getLimit(joint_name).max_position) ) );
+                                                   const double &joint_position) const {
+  bool limit_ok = (!(hasLimit(joint_name)
```
Comment (ct2034) [inline pilz_trajectory_generation/src/joint_limits_container.cpp]: This is also a good contribution in terms of verbosity. But can you please separate into an extra pr.

File: pilz_robot_programming/src/pilz_robot_programming/commands.py [hunk 2]
```diff
@@ -24,7 +24,10 @@
 from pilz_msgs.msg import MoveGroupSequenceGoal, MotionSequenceItem
 from moveit_msgs.msg import (OrientationConstraint, MotionPlanRequest, JointConstraint, Constraints,
                              PositionConstraint, PlanningOptions)
+from moveit_msgs.msg import RobotState
```
Comment (ct2034) [inline pilz_robot_programming/src/pilz_robot_programming/commands.py]: Please combine these

**Review by ct2034 — CHANGES_REQUESTED** [review]
This looks like a great addition. Thanks
To merge it, we need a test, though. Here [test/integrationtests/tst_api_cmd_conversion.py](https://github.com/PilzDE/pilz_industrial_motion/blob/melodic-devel/pilz_robot_programming/test/integrationtests/tst_api_cmd_conversion.py) and here [test/acceptance_tests/acceptance_test_blending.py](https://github.com/PilzDE/pilz_industrial_motion/blob/melodic-devel/pilz_robot_programming/test/acceptance_tests/acceptance_test_blending.py)

**Comment by DanBmh** [comment]
Im not sure how to test my blend sequence test, because for me it seems that i need a pilz robot for this.

**Pull request closed without merging**